typescript/options.d.ts

```diff
@@ -350,6 +350,12 @@ export interface InitOptions<T = object> extends PluginOptions<T> {
    */
   debug?: boolean;
 
+  /**
+   * Filter function that allows changing the logged option value when `debug` is set to true.
```

Comment:
but only for the log statement while init()... maybe the function name needs to respect this?